api/db/migrations/20240910144200_certification_whitelist.js

```diff
@@ -0,0 +1,21 @@
+const TABLE_NAME = 'certification-centers';
+const COLUMN_NAME = 'isScoBlockedAccessWhitelist';
+
+const up = async function (knex) {
+  await knex.schema.table(TABLE_NAME, function (table) {
+    table
+      .boolean(COLUMN_NAME)
+      .defaultTo(false)
+      .comment(
+        'As of for now, when boolean is false, it means than the center is currently eligible to SCO access closure. If true, it means you are whitelist (not closed)',
```

Comment:
```
'As of now, the center is currently eligible to SCO access closure when the property is set to false. Otherwise, the center is whitelisted (not closed)',
```

Reply:
Done !